src/PKSim.Core/Snapshots/Mappers/ExpressionContainerMapper.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Threading.Tasks;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Extensions;
+using PKSim.Assets;
+using PKSim.Core.Model;
+using PKSim.Core.Services;
+
+namespace PKSim.Core.Snapshots.Mappers
+{
+   public class ExpressionContainerMapperContext
+   {
+      public IndividualMolecule Molecule { get; set; }
+      public ISimulationSubject SimulationSubject { get; set; }
+   }
+
+   public class ExpressionContainerMapper : SnapshotMapperBase<MoleculeExpressionContainer, ExpressionContainer, ExpressionContainerMapperContext>
+   {
+      private readonly ParameterMapper _parameterMapper;
+      private readonly ITransportContainerUpdater _transportContainerUpdater;
+      private readonly MembraneLocation _defaultMembraneLocation;
+
+      public ExpressionContainerMapper(ParameterMapper parameterMapper, ITransportContainerUpdater transportContainerUpdater)
+      {
+         _parameterMapper = parameterMapper;
+         _transportContainerUpdater = transportContainerUpdater;
+         _defaultMembraneLocation = MembraneLocation.Basolateral;
```

Comment:
The Membrane is the one used the most in the database so I am not saving this value.